pkg/users/local.js

```diff
@@ -97,14 +122,16 @@ function AccountsPage() {
                 groups={groupsExtraInfo || []}
                 isGroupsExpanded={isGroupsExpanded}
                 setIsGroupsExpanded={setIsGroupsExpanded}
+                min_gid={min_gid}
+                max_gid={max_gid}
             />
         );
     } else if (path.length === 1) {
         return (
             <AccountDetails accounts={accountsInfo} groups={groupsExtraInfo} shadow={shadow || []}
                             current_user={current_user_info && current_user_info.name} user={path[0]} />
         );
-    }
+    } else return null;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18486-20230313-145525-cf69ea11-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)